web/cypress/e2e/overview/overview.spec.ts

```diff
@@ -22,7 +22,11 @@ describe('netflow-overview', () => {
     cy.get('#overview-panels-modal').contains('Select all').click();
 
     //Save
+    cy.setupNetworkIdleTracking('GET', '/api/flow/metrics*');
     cy.get('#overview-panels-modal').contains('Save').click();
+
```

Comment:
I am curious, how this is different than simply intercepting for `/api/flow/metrics` request and waiting for call to happen, like we have here:

https://github.com/openshift/openshift-tests-private/blob/master/frontend/views/netflow-page.ts#L5-L7